ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/434__data-science-pipelines-ui.robot

```diff
@@ -60,7 +60,7 @@ Verify User Can Create And Run A DS Pipeline From DS Project Details Page    # r
     ...    pipeline_name=${PIPELINE_TEST_NAME}
     Verify Pipeline Run Deployment Is Successful    project_title=${PRJ_TITLE}
     ...    workflow_name=${workflow_name}
-
+    Delete Pipeline Ran    ${PIPELINE_TEST_RUN_BASENAME}    ${PIPELINE_TEST_NAME}
```

Comment:
I believe the test case to be completed should delete also the other items of a pipeline, not only the run: the pipeline definition and the pipeline server